### PR TITLE
Add reverse-string exercise

### PR DESCRIPTION
**Pull request by glennj**
brian d foy writes about [perl's handling of grapheme clusters](https://www.effectiveperlprogramming.com/2011/06/treat-unicode-strings-as-grapheme-clusters/).

### CODE REVIEW

**Comment by glennj** [comment]
Ah, but further reading reveals `\X` regex shortcut: https://www.perl.com/pub/2012/05/perlunicookbook-extract-by-grapheme-instead-of-codepoint-regex.html/

**Review by kotp — APPROVED** [review]

**Comment by glennj** [comment]
I'll add some words about grapheme clusters. Maybe in the hints.

File: exercises/practice/reverse-string/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ignore
```
Comment (kotp) [inline exercises/practice/reverse-string/.docs/instructions.append.md:1]: Why the heading of ignore?  If it is a header, it should also be capitalized, I would think.

Comment (glennj) [reply exercises/practice/reverse-string/.docs/instructions.append.md:1]: markdown files have to have an H1, but it is ignored for append files. Just a quirk of exercism markdown standards.

**Pull request merged**